pdc-contact-browser.spec

```diff
@@ -28,6 +28,10 @@ cp serversetting.json %{buildroot}/var/www/html/%{name}
 /var/www/html/%{name}
 
 %changelog
+* Thu Dec 22 2016 Chuang Zhang <chuzhang@redhat.com> 1.2.0-1
+- new package built with tito
+- fix bugs
+- add assets for packaging
```

Comment:
Ideally there should be an empty line here to be consistent with rest of the changelog